tests/framework/extensions/kubeapi/rbac/create.go

```diff
@@ -68,3 +69,47 @@ func CreateRoleBinding(client *rancher.Client, clusterName, roleBindingName, nam
 
 	return newRoleBinding, nil
 }
+
+// CreateGlobalRole is a helper function that uses the dynamic client to create a global role in the local cluster.
+func CreateGlobalRole(client *rancher.Client, clusterName string, globalRole *v3.GlobalRole) (*v3.GlobalRole, error) {
```

Comment:
yes, makes sense to have it as a constant instead of having it as a parameter here. I will update CRTB functions as well since this comment applies to CRTBs as well.